docs/Ways-of-Working/Onboarding/Introduction.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Introduction to Onboarding at AM
+---
+
+## Welcome to AM
+
+We're delighted to have you on board. Our structured onboarding process is designed to facilitate a smooth and enriching transition into our team.
+
+Please note, if you're in a Lead role, you'll have additional responsibilities.
+
+## Onboarding Stages
+
+Our onboarding process is structured into four distinct stages:
+
+### 1. **Arrival**
+
+- **Duration**: First few hours.
+- **Objective**: Familiarize you with the AM environment and ensure a warm welcome.
```

Comment:
Love the objectives. I think this provides a really positive foundation to the onboarding process. I would consider putting this as a subheading or emphasised text right below the phase heading.

Reply:
Perhaps using the success or info admonition: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:success